Lab5.cpp

```diff
@@ -0,0 +1,28 @@
+﻿#include <iostream> 
+#include <string>
+#include "Methods.h"
+using namespace std;
+#define ARRAY_SIZE 3    
+
+int main() {
+    Shop a;
+    int input;
+    float money;
+    while (1) {
+        cout << "Best 1, Sort 2, Exit 0 \n inp:";
+        cin >> input;
+        switch (input)
+        {
```

Comment:
You use short non-informative changes.

You should:
switch (number_of_action_to_be_performed) {